definitions.yaml

```diff
@@ -24,7 +24,7 @@ definitions:
             description: Unique name of the installation
             type: string
           provider:
-            description: The technical provider used in this installation. Either "kvm" or "aws".
+            description: The technical provider used in this installation. Either "kvm", "aws" or "azure".
```

Comment:
I think we are missing an oxford comma here: `"kvm", "aws", or "azure"`

Reply:
Thanks, I learned something today :)